Bottom: d5432efc2e772754920ff314583f437b326bc298
Top:    ed7a5a27c08bceaff18c6df55fa715e8dc8ca6f5
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-20 09:51:00 -0500

Update configuration for running behind reverse proxy



---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index efeb94099..fb231c7cc 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -51,7 +51,7 @@ container_env:
   BUILDBOT_SMTP_USER: 'buildbot'
   BUILDBOT_SMTP_PASS: '{{ buildbot_email_password }}'
   BUILDBOT_WEB_PORT: '{{buildbot_http_port}}'
-  BUILDBOT_WEB_URL: 'http://{{ buildbot_hostname }}/'
+  BUILDBOT_WEB_URL: 'https://{{ buildbot_hostname }}/'
   BUILDBOT_WORKER_PORT: '{{buildbot_worker_port}}'
   BUILDBOT_WORKER_NTEL: 'worker-ntel'
   BUILDBOT_WORKER_NTEL_PASS: '{{ buildbot_worker_ntel_password }}'
